src/lib/utils/index.ts

```diff
@@ -43,8 +42,8 @@ export function getOauthTokens(proxyId: string, scopes: string[] | undefined): P
         : (['domoapps']);
 
       return Promise.all([
-        keytar.getPassword(`domoapps-oauth-access-${allScopes.join('-')}`, `${instance}-${proxyId}`),
```

Comment:
Won't these just return the same thing now? Before it was the OAuth access token and OAuth refresh token. Now there doesn't seem like a unique way to distinguish between them.

Reply:
I've resolved this. However, I'm just realizing that there's a "scopes" part of the service itself (what it's named in the credential manager). What function does that serve? I've only used the domoapps scope, but I'm curious without keytar if I need to account for another scope or if it can just be saved as is

Reply:
Good catch. When you request a token and refresh token they are tied to the scopes you have requested. We don't want a token with the wrong scopes returned. We need a way store different tokens for the same instance but with different scopes. Let me know if this needs further clarification.